src/GrumPHP/Task/Phpcsfixer.php

```diff
@@ -71,7 +71,7 @@ public function run(ContextInterface $context)
         }
 
         if ($config['config_file']) {
-            $this->processBuilder->add('--config-file' . $config['config-file']);
+            $this->processBuilder->add('--config-file' . $config['config_file']);
```

Comment:
Shouldn't there also be an equal sign between the option and the value? Like this: `config-flle=path`